package.json

```diff
@@ -100,8 +100,7 @@
     "extensionizer": "^1.0.0",
     "fast-json-patch": "^2.0.4",
     "fast-levenshtein": "^2.0.6",
-    "fuse.js": "^3.1.0",
-    "gulp": "github:gulpjs/gulp#4.0",
```

Comment:
I'd usually ask for dependency cleanups to be in separate PRs, so they don't mire up discussion over a proposal.

Reply:
Oooh, yeah sorry I didn't catch that in my commit diffs. I must have accidentally updated the project deps with NPM when I was setting up the project. I can revert those changes.